Description: Allow any version of pytz.
 Debian pytz package (ie: python-tz) is heavily patched and up-to-date, there's
 no reason why we should require such a high version.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-09-04

--- python-oslo.utils-2.4.0.orig/requirements.txt
+++ python-oslo.utils-2.4.0/requirements.txt
@@ -13,7 +13,7 @@ six>=1.9.0
 iso8601>=0.1.9
 oslo.i18n>=1.5.0 # Apache-2.0
 monotonic>=0.3 # Apache-2.0
-pytz>=2013.6
+pytz
 netaddr!=0.7.16,>=0.7.12
 netifaces>=0.10.4
 debtcollector>=0.3.0 # Apache-2.0
